tests/test_bad_fonts_nonrecursive.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
This script is the same as test_bad_fonts.sh except that it does not recursively enter directories inside ./tests/fonts/bad, and therefore does not test graphite fonts, which we do not expect a non-Graphite build to catch.